src/nopenal.h

```diff
@@ -10,8 +10,8 @@
 
 #include "ncompat.h"
 
-#include "AL/alc.h"
```

Comment:
Looking at openal soft examples, it does seem like `"AL/*.h"` is the proper way to include them (https://github.com/kcat/openal-soft/blob/master/examples/alhrtf.c)

Reply:
Apparently, the reason I changed this is because the mac build fails when AL/ is added.

Reply:
With autotools it works with "AL/" though, is this some difference in implementation?

Reply:
It turned out this was because Meson was picking up OpenAL.framework. It should stop doing that and use openal-soft now.